spec/classes/screensaver_spec.rb

```diff
@@ -4,22 +4,30 @@
 context 'supported operating systems' do
     on_supported_os.each do |os, os_facts|
       context "on #{os}" do
-        let(:facts) { os_facts.merge( { :gdm_version => '3.20.1' } ) }
+        let(:facts) do
```

Comment:
I could see it go either way in terms of clarity, but this approach seems inconsistent with the same fact as it's set in [L22](https://github.com/simp/pupmod-simp-gnome/pull/17/files#diff-b68f346c1d243bd3c82f3791ac4f4240R22) and [L27](https://github.com/simp/pupmod-simp-gnome/pull/17/files#diff-b68f346c1d243bd3c82f3791ac4f4240R27)